app/views/contexts/list_collector.py

```diff
@@ -2,57 +2,64 @@
 from flask_babel import lazy_gettext
 
 from app.views.contexts.question import build_question_context
-from app.questionnaire.placeholder_transforms import PlaceholderTransforms
 
 
-def build_list_collector_context(rendered_block, form, list_store, answer_store):
-    question_context = build_question_context(rendered_block, form)
-
+def build_list_items_summary_context(rendered_block, answer_store, list_store):
     list_name = rendered_block['for_list']
     list_item_ids = list_store[list_name].items
 
+    try:
+        item_titles = rendered_block.get('summary', {})['item_title']
+        item_titles = item_titles.strip('[]').replace("'", '').split(',')
```

Comment:
I also don't understand what this line is for? Is it escaping the title?

Reply:
Rendered blocks contain the titles as strings in the following format:
`"['a a', 'b b', 'c c']"`